forge/src/main/java/org/openjdk/skara/forge/HostedRepository.java

```diff
@@ -82,4 +83,29 @@ default PullRequest createPullRequest(HostedRepository target,
                                           List<String> body) {
         return createPullRequest(target, targetRef, sourceRef, title, body, false);
     }
+
+    default URI reviewUrl(Hash hash) {
+        var comments = this.commitComments(hash);
+        var reviewComment = comments.stream().filter(
+                c -> c.body().startsWith("<!-- COMMIT COMMENT NOTIFICATION -->")).findFirst();
+
+        if (reviewComment.isEmpty()) {
+            return null;
+        }
+
+        /** The review comment looks like this:
+         * <!-- COMMIT COMMENT NOTIFICATION -->
+         * ### Review
+         *
+         * - [openjdk/skara/123](https://git.openjdk.java.net/skara/pull/123)
+         */
+
+        var pattern = Pattern.compile("### Review[^]]*]\\((.*)\\)");
```

Comment:
Do you need to use `Pattern.DOTALL` here? As in `Pattern.compile("### Review[^]]*]\\((.*)\\)", Pattern.DOTALL);` ? I'm thinking about the `.*` handling `\n`.

Reply:
As you can see from the test, this works just fine. :-)

The `DOTALL` flag only applies to the dot `.`. I'm searching for "anything but ]", `[^]]` which also includes line terminators. And the URL (that I match with `.*`) should of course not have line breaks in it.